plugins/optimization-detective/optimization.php

```diff
@@ -233,6 +233,11 @@ function od_optimize_template_output_buffer( string $buffer ): string {
 			continue;
 		}
 
+		// Elements in the Admin Bar are not relevant for optimization, so skip passing them to tag visitors.
+		if ( $processor->is_admin_bar() ) {
+			continue;
+		}
```

Comment:
Instead of calling this here, could this be baked into the tag processor class itself? For instance, as long as we're in the admin bar, the tag processor could move on to the next element.

I think that would be better for ergonomics, as it would handle the issue in one place and would avoid having to introduce a public method for something as specific as this.

Reply:
Yeah, I think that makes sense. See 8373c353d.